lib/pbench/server/api/resources/query_apis/month_indices.py

```diff
@@ -1,92 +1,36 @@
-from logging import Logger
-
 from flask import jsonify
-from flask_restful import Resource, abort
-import requests
+from logging import Logger
+from typing import Any, AnyStr, Dict
 
 from pbench.server import PbenchServerConfig
-from pbench.server.api.resources.query_apis import get_es_url, get_index_prefix
+from pbench.server.api.resources.query_apis import ElasticBase, Schema
 
 
-class MonthIndices(Resource):
+class MonthIndices(ElasticBase):
     """
     Get the range of dates in which datasets exist.
     """
 
     def __init__(self, config: PbenchServerConfig, logger: Logger):
-        """
-        __init__ Initialize the resource with info each call will need.
+        super().__init__(config, logger, Schema())
```

Comment:
I found the answer about [Black and trailing commas](https://black.readthedocs.io/en/stable/the_black_code_style.html#the-magic-trailing-comma):

>Early versions of Black used to ruthlessly collapse those into one line (it fits!). Now, you can communicate that you don’t want that by putting a trailing comma in the collection yourself. When you do, Black will know to always explode your collection into one item per line.
\
How do you make it stop? Just delete that trailing comma and Black will collapse your collection into one line if it fits.